x/wasm/handler.go

```diff
@@ -40,7 +40,12 @@ func NewHandler(k Keeper) sdk.Handler {
 }
 
 func handleStoreCode(ctx sdk.Context, k Keeper, msg *MsgStoreCode) sdk.Result {
-	codeID, err := k.Create(ctx, msg.Sender, msg.WASMByteCode)
+	_err := msg.ValidateBasic()
```

Comment:
Why `_err`?
Custom is to use `sdkerr` for this case? (As `err` has different type, so you need two)